src/styles/blocks/page.scss

```diff
@@ -0,0 +1,8 @@
+.page {
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: 400;
+
+  &__body {
+    margin: 0;
+  }
+}
```

Comment:
You can use `html` or `body` tag for it and rename this file. It will now be `reset.scss` instead of `page.scss`.